WaterLog_Backend/Controllers/SegmentLeaksController.cs

```diff
@@ -86,6 +86,16 @@ public async Task<ActionResult<SegmentLeaksEntry>> Get(int id)
             return leaks;
         }
 
+
+        // GET api/segment
+        [HttpGet("segment/{Id}")]
+        public async Task<ActionResult<IEnumerable<SegmentLeaksEntry>>> GetSegmentHistory(int Id)
+        {
+            return await _db.SegmentLeaks.Where( row => row.SegmentsId == Id ).ToListAsync();
+
+        }
+
+
```

Comment:
Remove double whitespace, should only be single between methods

Reply:
Fixed